libs/design/switch/examples/src/basic-switch/basic-switch.component.html

```diff
@@ -1,3 +1,3 @@
-<daff-switch [checked]="checked">Default</daff-switch>
+<daff-switch [checked]="checked" [formControl]="defaultSwitchExample" ngDefaultControl>Default</daff-switch>
 <daff-switch [checked]="checked" disabled>Disabled</daff-switch>
```

Comment:
Can you create separate examples for disabled and loading? There should be another example to show switch with an error message too.